src/app/(auth)/account/analysis-upload-log/page.tsx

```diff
@@ -19,6 +22,12 @@ export default async function AnalysisUploadLog(props: {
 }) {
   const t = await getTranslations('AnalysisUploadLog');
 
+  const user = await getUsersMe();
+
+  if (!isAnalysist(user.role)) {
+    redirect('/not-found');
```

Comment:
in server components you have function notFound()

Reply:
fixed